apps/docs/src/common/pages/get-started/use-prototype.tsx

```diff
@@ -51,7 +84,7 @@ const Page: FC<PageProps> = ({ location }) => (
           </div>
         </div>
         <p></p>
-        <p>2. Open up your prototype in a text editor like Atom. Find the <code>index.tsx</code> file in <code>src/common/pages/index.tsx</code> and paste the code between the <code>fragment</code>  tags. Save your changes.</p>
+        <p>2. Open up your prototype in Atom and put the code in <code>index.html</code> or file of your choice. Remove the additional code outside of <code>fragment</code> tags (this was only useful when this was an <code>index.tsx</code> file). Paste the selected component code within the <code>fragment</code> tags. Save your changes.</p>
```

Comment:
I wonder if we should just tell people to delete the contents of the file and then to paste in any HTML code they want to use.

Reply:
Sounds good and more concise. I've rephrased it to: Open your prototype in Atom and find index.html. Delete the contents of the file and paste in any HTML code you want to use between the fragment tags. Save your changes.